mitodl/ocw-to-hugo
# AWS Download Functionality

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-to-hugo/issues/41

#### What's this PR do?
The optional `-c` argument has been added, which takes the location of a JSON file containing a structure detailed in the README.  The courses listed in this JSON file are iterated and downloaded to the source folder passed in at `-s`.

#### How should this be manually tested?
Tests should pass, and you should be able to follow the instructions in the README to successfully connect to an `open-learning-course-data` bucket and download a list of courses which are subsequently converted to markdown.
